### ISSUE TITLE
New Lights

### ISSUE THREAD

**Issue by lrivard00** [Electrical-Age/ElectricalAge#449]
Could we have more lights? One like you find that hang in modern office buildings? ex. 
(https://cloud.githubusercontent.com/assets/18453476/15765352/884a4150-28f9-11e6-8df2-7bd0793e46fa.png)
Or more fancy ones like... ex. 
(https://cloud.githubusercontent.com/assets/18453476/15765372/cbb13a02-28f9-11e6-8a9e-3f61f34d80b2.png)

**Comment by mattmtgaming005** [comment]
nice

**Comment by Baughn** [comment]
The quickest way to get more light models is if you supply the models. :)

I think we have enough that it's worth focusing on other things, but adding a new model to the game is easy enough.

**Comment by AMIDIBOSS** [comment]
I can make some fancy models if you tell me how (file format and model parameters)

**Comment by cm0x4D** [comment]
We use blender as modeling tool and we export to Wavefront format (.obj) for the mod. There is a template to start with [here](https://github.com/Electrical-Age/ElectricalAge/blob/develop/src/main/resources/assets/eln/model/_TEMPLATES/transparentNode.blend) for transparent nodes (multiple blocks) and [here](https://github.com/Electrical-Age/ElectricalAge/blob/develop/src/main/resources/assets/eln/model/_TEMPLATES/sixNode.blend) for six-node blocks.

**Comment by AMIDIBOSS** [comment]
Okay, I'll upload models to my google drive if that's not a problem. What is the first task?

**Comment by cm0x4D** [comment]
Have a look in the issues and pick something you would like to do. Best would be a new model for some existing stuff like new lamps...

**Comment by AMIDIBOSS** [comment]
OK, i'm making first 2 models already, but how do i make on/off state? are these animation states?

**Comment by cm0x4D** [comment]
No. In the most cases we display some parts of the model and some not depending the state. Sometimes we move or rotate parts depending the state, but all this is done in code. And it is a lot of work and not trivial. So the less moving parts, the better. Rendering parts or not is pretty simple tough.

**Comment by AMIDIBOSS** [comment]
I tried to download some of existing models and here's what i found:
The models are textured from separate PNG file, having on/off states two images

**Comment by cm0x4D** [comment]
Yes, that works too. Depending what you want to achieve the one or the other is simpler. I had the most time just to add a status LED or alike, so the solution with multiple parts was simpler...
